<launch>

        <!-- <node pkg=" package name " 
            name="node name" 
            type="file name"
            output="screen">
        </node> -->


        <!-- <node pkg="hebi_useful_pkg" 
            name="hebi_control" 
            type="hebi_control"
            output="screen">
        </node> -->

        <node pkg="hebi_useful_pkg" 
            name="m3_joy" 
            type="m3_joy"
            output="screen">
        </node>
        

        <node pkg="hebi_useful_pkg" 
            name="joy_pubsub_node" 
            type="joy_pubsub_node"
            output="screen">
        </node>
        
        <node pkg="hebi_useful_pkg" 
            name="steer_rotation_pub" 
            type="steer_rotation_pub"
            output="screen">
        </node>


        <node pkg="joy" 
            name="joy_node" 
            type="joy_node"
            
            output="screen">
            <param name="dev" type="string" value="/dev/input/js0"/>
        </node>

</launch>